### PR TITLE
Changelog 2.2.12

### PR DESCRIPTION
**Pull request by drew-gross**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 ## Parse Server Changelog
 
+### 2.2.12 (9/9/2016)
```
Comment (neophob) [inline CHANGELOG.md]: 9/9/2016 sounds like this commit is from Marty McFly

**Comment by facebook-github-bot-9** [comment]
@drew-gross updated the pull request.

**Comment by facebook-github-bot-6** [comment]
@drew-gross updated the pull request.

**Comment by drew-gross** [comment]
Reviewed in slack

**Pull request merged**